components/Newsletter/NewsletterEmailSubscribe.tsx

```diff
@@ -0,0 +1,109 @@
+import Box from "components/Box";
+import Flex, { FlexProps } from "components/Flex";
+import styled from "styled-components";
+import css from "@styled-system/css";
+import { transition } from "components/system";
+import { useMarketo } from "utils/marketo";
+
+const { NEXT_PUBLIC_EMAIL_SUBSCRIPTION_FORM_ID } = process.env;
+
+export function NewsletterEmailSubscribe({ ...props }: FlexProps) {
+  const formLoaded = useMarketo({
+    formId: NEXT_PUBLIC_EMAIL_SUBSCRIPTION_FORM_ID,
+  });
+
+  return (
+    <Flex
+      backgroundColor="white"
+      justifyContent="center"
+      alignItems="stretch"
+      flexDirection={["column", "row"]}
```

Comment:
```suggestion
      alignItems="stretch"
      flexDirection="column"
```

Nit: Need to test in actual browser, but this way you should be able to remove `minWidth` from the child element. Also you can remove child `Box` entirely and move all its props to this `Flex`.

Reply:
Removing minWidth doesn't work - we'll have to revisit this another time.

Reply:
Removing minWidth breaks at various resolutions.  Will need to revisit this later.